dissect/target/plugins/os/windows/log/evtx.py

```diff
@@ -91,16 +94,19 @@ def _parse_chunk(self, _, chunk: bytes) -> Iterator[Record]:
         chnk = evtx.ElfChnk(chunk)
         try:
             for event in chnk.read():
-                yield self._build_record(event)
+                yield self._build_record(event, None)
         except MalformedElfChnkException:
             pass
 
-    def _build_record(self, evtx_record) -> Record:
+    def _build_record(self, evtx_record: dict, source: Path) -> Record:
         # predictable order of fields in the list is important, since we'll
         # be constructing a record descriptor from it.
         evtx_record_fields = sorted(evtx_record.items())
 
-        record_values = {"_target": self.target}
+        record_values = {
+            "_target": self.target,
+            "evtx_source": source,
```

Comment:
```suggestion
            "source": source,
```
Isn't `source` more widely used in other plugins?

Reply:
Maybe handle that in the loop below? Like adding a `_` at the end if there's a conflict with an existing field name?

Theoretically `ts`, `evtx_source`, `hostname` and `domain` could be key names too 😄 and we don't actually deal with that either.